packages/harmony/src/storybook/components/SidebarItem.tsx

```diff
@@ -22,19 +19,25 @@ export const SidebarItem = (props: SidebarItemProps) => {
       : theme.barSelectedColor
 
   return (
-    <Flex justifyContent='space-between' w='100%'>
-      <Box as='div'>{name}</Box>
-      <Flex
-        justifyContent='center'
-        alignItems='center'
-        as='div'
-        ph='s'
-        borderRadius='s'
-        border='default'
-        css={{ height: '18px', borderColor: statusColor, color: statusColor }}
+    <div
+      css={{ display: 'flex', justifyContent: 'space-between', width: '100%' }}
+    >
+      <span>{name}</span>
```

Comment:
Looks like the Box was a div before, not a span. Not sure if that matters here?

Reply:
yeah i checked, seemed to not matter